keyboards/yunzii/al68/al68.h

```diff
@@ -0,0 +1,23 @@
+// /* Copyright 2024 Jacky
+//  *
+//  * This program is free software: you can redistribute it and/or modify
+//  * it under the terms of the GNU General Public License as published by
+//  * the Free Software Foundation, either version 2 of the License, or
+//  * (at your option) any later version.
+//  *
+//  * This program is distributed in the hope that it will be useful,
+//  * but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  * GNU General Public License for more details.
+//  *
+//  * You should have received a copy of the GNU General Public License
+//  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+//  */
+
+enum custom_keycodes {
```

Comment:
It may be worth retargeting to develop, and using the internal keycodes: 
https://github.com/qmk/qmk_firmware/blob/6af8d4210c35e42643d32fcba806885e6b6efab4/quantum/keycodes.h#L625-L638

Reply:
Thanks .  I will use BLE custom driver when develop branch internal ble keycode merge to master.